fili-core/src/main/java/com/yahoo/bard/webservice/logging/LogFormatterProvider.java

```diff
@@ -0,0 +1,53 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.logging;
+
+import com.yahoo.bard.webservice.config.SystemConfig;
+import com.yahoo.bard.webservice.config.SystemConfigProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A class to provide a log formatter instance.
+ */
+public class LogFormatterProvider {
+
+    private static final Logger LOG = LoggerFactory.getLogger(LogFormatterProvider.class);
+    private static final SystemConfig SYSTEM_CONFIG = SystemConfigProvider.getInstance();
+
+    private static final String DEFAULT_LOG_FORMATTER_IMPL = JsonLogFormatter.class.getCanonicalName();
+
+    private static final String LOG_FORMATTER_IMPL_KEY = SystemConfigProvider.getInstance().getStringProperty(
+            SYSTEM_CONFIG.getPackageVariableName("log_formatter_impl"),
```

Comment:
Can we expand this to `implementation`? In general, we try to avoid abbreviations if we can so that the code is more understandable and clear.